server/src/like/like.controller.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { getModelToken } from '@nestjs/mongoose';
+import { Test, TestingModule } from '@nestjs/testing';
+import { User } from 'src/user/entities/user.entity';
```

Comment:
```suggestion
import { Test, TestingModule } from '@nestjs/testing';

import { User } from 'src/user/entities/user.entity';
```

외부모듈 import 와 내부 모듈 import 사이 개행이...!